cmd/promtool/testdata/prometheus-config.lint.yml

```diff
@@ -1,2 +1,3 @@
-rule_files:
-  - prometheus-rules.lint.yml
```

Comment:
Is removing this intentional?  Was added in https://github.com/prometheus/prometheus/pull/10952

Reply:
Oh I see. 
I have rollbacked this file and created a new dedicated file for test.